spec/models/group_spec.rb

```diff
@@ -125,6 +125,22 @@
     context 'with bob in the team' do
       before { team.people << bob }
 
+
+      context 'and bob is a team leader' do
+
+        before do
+          bob.memberships.first.update(leader: true)
+        end
```

Comment:
I've changed the before block to not assign to `@leadership`. The before block now smaller, and matches the context description.